demos/image-classification/index.html

```diff
@@ -88,7 +88,8 @@ <h1 class="title">
         <div class="grid-2">
             <div>
                 <div id="container">
-                    <img id="image" src="./static/tiger.jpg" />
+                    <img id="image"
+                        src="./static/tiger.jpg" />
```

Comment:
```suggestion
                    <img id="image" src="./static/tiger.jpg" />
```
(minor) random new line break?

Reply:
Fixed :)